test/test.dart

```diff
@@ -206,16 +207,7 @@ Future<void> setupTests(List<String>? args) async {
       }
 
       for (final path in paths) {
-        try {
-          Realm.deleteRealm(path);
-        } catch (e) {
-          print("Can not delete realm at path: $path. Did you forget to close it?");
-        }
-        String pathKey = _path.basenameWithoutExtension(path);
-        String realmDir = _path.dirname(path);
-        await Directory(realmDir).list().forEach((f) {
```

Comment:
I prefer `await for` over `await x.forEach((y) {})` https://api.dart.dev/stable/2.16.2/dart-io/Directory-class.html

Reply:
That's deleted code though?